web/src/features/fba/components/ZoneSummaryPanel.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react'
 import { styled } from '@mui/material/styles'
 import CombustibleAreaViz from 'features/fba/components/viz/CombustibleAreaViz'
-import { Grid, Typography } from '@mui/material'
+import { Grid, IconButton, Typography } from '@mui/material'
 import { isUndefined } from 'lodash'
 import { ElevationInfoByThreshold, FireZone, FireZoneArea, FireZoneThresholdFuelTypeArea } from 'api/fbaAPI'
 import ElevationInfoViz from 'features/fba/components/viz/ElevationInfoViz'
 import FuelTypesBreakdown from 'features/fba/components/viz/FuelTypesBreakdown'
+import CloseIcon from '@mui/icons-material/Close'
 
 const SidePanelGrid = styled(Grid)({
   minWidth: 400,
-  overflowY: 'auto',
+  overflowY: 'scroll',
```

Comment:
This was changed because I believe with those stats the scroll will always be there anyway and it removes a little UI stutter when selecting a different fire zone